README.md

```diff
@@ -210,27 +92,18 @@ Example:
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/Theodoraldo)
+- Twitter: [@twitterhandle](https://twitter.com/AsimengOse33947)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/theodoreasimeng/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[Application to access vet db]**
+- [ ] **[Deployment]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly 😃as stated by the previous reviewer it will be better if you fix the links back to the top to let the users scroll up to the top of the readme quickly, so please try to solve that to make the readme professional you fix that by adding ``` <a name="readme-top"></a>``` at the top of the readme to let the scroll working 💯

Reply:
His references were on the issues link which I did.